com.unity.sg2/Editor/GraphUI/DataModel/ShaderGraphModel.cs

```diff
@@ -669,8 +670,21 @@ public static bool DoesNodeRequireTime(GraphDataNodeModel graphDataNodeModel)
             return nodeRequiresTime;
         }
 
+        // Temporarily hide some unfinished nodes: https://jira.unity3d.com/browse/GSG-1290
+        // Should have a feature for managing what types/nodes are exposed i nbuil
+        static readonly string[] blacklist = new string[] {
+            "CustomRenderTextureSelf",
+            "CustomRenderTextureSize",
+            "CustomRenderTextureSlice",
+            "ParallaxOcclusionMapping",
+            "LinearBlendSkinning",
+            "Reference"
+        };
+
         public bool ShouldBeInSearcher(RegistryKey registryKey)
         {
+            if (blacklist.Contains(registryKey.Name))
+                return false;
```

Comment:
I'm not proud of this-- but I don't want to introduce an architectural solution for hiding features in this PR. That's a bigger discussion.